src/artemis/__main__.py

```diff
@@ -51,17 +51,25 @@ class BlueskyRunner:
     current_status: StatusAndMessage = StatusAndMessage(Status.IDLE)
     last_run_aborted: bool = False
 
-    def __init__(self, RE: RunEngine) -> None:
+    def __init__(self, RE: RunEngine, skip_startup_connection=False) -> None:
         self.RE = RE
+        self.skip_startup_connection = skip_startup_connection
         if VERBOSE_EVENT_LOGGING:
             RE.subscribe(VerbosePlanExecutionLoggingCallback())
-        for plan in PLAN_REGISTRY:
-            PLAN_REGISTRY[plan]["setup"]()
+
+        if not self.skip_startup_connection:
+            for plan in PLAN_REGISTRY:
+                PLAN_REGISTRY[plan]["setup"]()
```

Comment:
Nit: Might be a little cleaner to pull these two lines into a function to call here and in `start` e.g. `setup_all_plans`